src/reference/90-Developers-Guide/09-Launcher/01-Launcher-Getting-Started.md

```diff
@@ -116,11 +122,14 @@ interface class will be provided by the launcher, so it is only a
 compile-time dependency. If you are building with sbt, your dependency
 definition would be:
 
-Make the entry point to your class implement 'xsbti.AppMain'. An example
+FIXME
```

Comment:
I pulled this out of [the 0.12 docs][1] where it must have been lost in translation somehow, and updated the version:

```scala
libraryDependencies += "org.scala-sbt" % "launcher-interface" % "1.0.0" % "provided"

resolvers <+= sbtResolver
```

Assuming that is still correct, I'll commit it to this branch.

[1]: http://www.scala-sbt.org/0.12.1/docs/Detailed-Topics/Launcher.html#creating-a-launched-application